extensions/objects/templates/fraction-editor.component.ts

```diff
@@ -51,6 +51,7 @@ export class FractionEditorComponent implements OnInit {
     if (newFraction.length === 0) {
       this.errorMessage = 'Please enter a non-empty fraction value.';
       this.currentFractionValueIsValid = false;
+      return;
```

Comment:
After setting the error message for an empty fraction it should stop running the function. Otherwise, it will check if the fraction has any errors in it's format